deployments/ansible/site.yml

```diff
@@ -0,0 +1,10 @@
+---
+# This playbook deploys a Signalfx auto-discovery agent
```

Comment:
Is this for testing or is it used in the actual usage of the module?

Reply:
This is actual usage of the module. Renamed to playbook.yml for better readability and to avoid confusion.